ibm-event-streams/README.md

```diff
@@ -165,19 +166,19 @@ and the Db2 JDBC 4 jar (installed as part of the Db2 driver package) to the Java
     /path/to/your/kafka_2.11-1.0.0/bin/connect-standalone.sh /path/to/your/kafka_2.11-1.0.0/config/connect-standalone.properties /path/to/your/connect-cloudant-source.properties /path/to/your/connect-jdbc-sink.properties
     ```
 1. The Kafka Connect worker will run
-    1. Reading documents from the Cloudant database and publishing them to the Message Hub topic
+    1. Reading documents from the Cloudant database and publishing them to the Event Streams topic
         * On stdout you will see a message like `INFO Return 20 records with last offset "..."`
         as `kafka-connect-cloudant` publishes messages to the topic.
-    1. Consuming messages from the Message Hub topic and inserting them into the Db2 Warehouse on Cloud table.
+    1. Consuming messages from the Event Streams topic and inserting them into the Db2 Warehouse on Cloud table.
 1. The worker will continue to run until a terminal error or it is stopped.
 1. [View the extracted data imported into the Db2 table](../db2/view_data.md).
 
 # References
 
 * [Apache Kafka Connect](https://docs.confluent.io/current/connect/userguide.html)
-* [IBM Cloud Message Hub with Apache Kafka Connect](https://console.bluemix.net/docs/services/MessageHub/messagehub113.html#kafka_connect)
+* [IBM Event Streams with Apache Kafka Connect](https://cloud.ibm.com/docs/services/EventStreams?topic=eventstreams-kafka_connect)
 * [`kafka-connect-cloudant`](https://github.com/cloudant-labs/kafka-connect-cloudant)
 * [`kafka-connect-jdbc`](https://github.com/confluentinc/kafka-connect-jdbc)
 * [Confluent JDBC connector documentation](https://docs.confluent.io/current/connect/connect-jdbc/docs/index.html)
-* [Creating a Kafka topic with the Message Hub admin REST API](https://github.com/ibm-messaging/message-hub-docs/tree/master/admin-rest-api#creating-a-kafka-topic)
-* [Connecting to Db2 Warehouse with JDBC](https://www.ibm.com/support/knowledgecenter/SS6NHC/com.ibm.swg.im.dashdb.doc/connecting/connect_connecting_jdbc_applications.html)
+* [Creating a Kafka topic with the Event Streams admin REST API](https://github.com/ibm-messaging/message-hub-docs/tree/master/admin-rest-api#creating-a-kafka-topic)
```

Comment:
This link still works, but it appears to redirect to https://github.com/ibm-messaging/event-streams-docs/tree/master/admin-rest-api#creating-a-kafka-topic so maybe we should just update it anyway.

Reply:
Fixed in ee9fdf1